src/System.Drawing.Common/tests/Printing/PrintDocumentTests.cs

```diff
@@ -257,7 +257,7 @@ private void AssertDefaultPageSettings(PageSettings pageSettings)
             }
 
             Assert.False(pageSettings.Landscape);
-            Assert.Equal(PaperSourceKind.FormSource, pageSettings.PaperSource.Kind);
+            Assert.True(pageSettings.PaperSource.Kind == PaperSourceKind.FormSource || pageSettings.PaperSource.Kind == PaperSourceKind.AutomaticFeed);
```

Comment:
It sounds like that these values can be anything depending on the system configuration.

Should these validations be rather deleted; or the tests moved to outer loop?

Reply:
Yeah, since the "default" actually looks up the current system setting, and since that depends on what printer someone has plugged in, this is fairly brittle; I suggest it should be deleted.